src/share/classes/jdk/jfr/internal/Options.java

```diff
@@ -139,6 +143,27 @@ public static synchronized boolean getSampleThreads() {
         return sampleThreads;
     }
 
+    public static synchronized void setSampleObjectAllocations(Boolean sample) {
+        jvm.setSampleObjectAllocations(sample);
+        sampleObjectAllocations = sample;
+    }
+
+    public static synchronized boolean getSampleObjectAllocations() {
+        return sampleObjectAllocations;
+    }
+
+    public static synchronized void setObjectAllocationsSamplingInterval(Long interval) {
+        if (interval <= 0) {
```

Comment:
I'm assuming tiny interval is harmful, such as once every 1 ms, does it make sense to set a lower bound?
otherwise, I guess the test should be refined to cover tiny intervals.

Reply:
the unit of the Parameter "interval" is count.
If the user sets it to 1,  that's means every object allocated in c2 will be sampled.

Reply:
needs some comments to explain interval is count not time.